src/frontend/src/btc/components/transactions/BtcTransactions.svelte

```diff
@@ -0,0 +1,22 @@
+<script lang="ts">
+	import { slide } from 'svelte/transition';
+	import BtcTransaction from '$btc/components/transactions/BtcTransaction.svelte';
+	import {
+		sortedTransactions,
+		transactionsNotInitialized
```

Comment:
In another PR can you prefix those derived store with `btc`?

We got now two derived stores called `transactionsNotInitialized` within the all app.

Reply:
Sure, noted.